scala/build/Dockerfile.maintenance

```
@@ -0,0 +1,259 @@
+
+# (Optional) Install XVFB if there's a need to run browsers in headful mode
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    xvfb
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    libc6
+
+# === INSTALL Node.js ===
+
+# Install node 20.x
+RUN apt-get update && apt-get install -y curl && \
```

Hadolint

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)